recipes/celero/all/patches/cmake-install-pic-cxx-standard-sytem-libs.patch

```diff
@@ -0,0 +1,86 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -24,6 +24,9 @@
+ CMAKE_MINIMUM_REQUIRED(VERSION 3.0.2)
+ 
+ macro(CeleroSetDefaultCompilerOptions)
++	set_target_properties(${PROJECT_NAME} PROPERTIES
```

Comment:
Is it really required? The fPIC option is converted to -DPOSITION_INDEPENDENT_CODE=ON/OFF by Conan automatically. I have the impression this could be removed.

Reply:
It comes from @madebr PR in celero repo, so if it is accepted, this option will be required in the recipe, otherwise it will override conan PIC injection.

Reply:
Although I have not received any feedback on my celero pr, this would make it have the same behavior.